clients/src/main/resources/common/message/FindCoordinatorRequest.json

```diff
@@ -27,7 +27,9 @@
   // Version 4 adds support for batching via CoordinatorKeys (KIP-699)
   //
   // Version 5 adds support for new error code TRANSACTION_ABORTABLE (KIP-890).
-  "validVersions": "0-5",
+  //
+  // Version 6 adds support for share groups (KIP-932).
```

Comment:
We might want to update KIP with version 6 as KIP says version 5 is added for KIP-932.

Reply:
Yes, I will do a KIP update once all of the IDs are confirmed by the merge. Essentially, KIP-890 overtook KIP-932.

Reply:
Done.